### PR TITLE
Viar/fix scenario polling on workspace change prod 11537

### PR DESCRIPTION
**Pull request by ardvi**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/state/commons/ScenarioConstants.js [hunk 1]
```diff
@@ -22,4 +22,5 @@ export const SCENARIO_ACTIONS_KEY = {
   SET_SCENARIO_NAME: 'SET_SCENARIO_NAME', // Use to update reducer by setting scenario name in scenarios list
   START_SCENARIO_STATUS_POLLING: 'START_SCENARIO_STATUS_POLLING',
   STOP_SCENARIO_STATUS_POLLING: 'STOP_SCENARIO_STATUS_POLLING',
+  STOP_ALL_SCENARIO_STATUS_POLLING: 'STOP_ALL_SCENARIO_STATUS_POLLING',
```
Comment (esasova) [inline src/state/commons/ScenarioConstants.js:25]: just a question: is it an action stopping polling of all scenarios ? If yes, typo: STOP_ALL_SCENARIO**S**_STATUS_POLLING :slightly_smiling_face:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by esasova — APPROVED** [review]
One question, LGTM :+1:

File: src/state/commons/ScenarioConstants.js (hunk 1 above)
Comment (ardvi) [reply src/state/commons/ScenarioConstants.js]: yes modified to STOP_ALL_SCENARIO_STATUS_POLLING

**Pull request merged**